autogen/main/variables.tf.tmpl

```diff
@@ -406,6 +406,7 @@ variable "enable_binary_authorization" {
 }
 
 variable "pod_security_policy_config" {
+  type = list(map(string))
```

Comment:
I think an object would be better actually.

```suggestion
  type = list(object({ enabled = bool }))
```

Reply:
I just updated the PR accordingly, wasn't sure about how strong the constraint should be :)